sagemaker-pipelines/step-decorator/local-mode-example/using_step_decorator_with_local_mode.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>We don't need to initialize the <code>role = None</code>.Rather, simply combine the three cells to:</p><pre class="ql-syntax" spellcheck="false">import sagemaker
role = sagemaker.get_execution_role() 
print(role)
</pre><p> </p><p>For one thing, the auto retrieval of default roles defined in the config file does not happen here. It happens in pipeline compile time (i.e. generating pipeline json definition)</p><p>For another, the <code>role=None</code> and <code>if role is None</code> condition check are redundant because role is None for sure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>